docs/gc.md

```diff
@@ -83,17 +76,17 @@ The *nursery* area is subdivided into two further areas, the *allocate* space an
 3. A local GC scavenge process runs and reachable objects are either copied into the *survivor* space or into the *tenure* area if they have reached *tenure age*. Any objects that can't be reached are left untouched and subsequently cleared.
```

Comment:
This is mostly terminology thing, but allocate space during GC time is called evacuate. It may not be that interesting for STW variant to mention (while we still consider it evacuate, it's not really exposed), but when/if we elaborate concurrent variant, some part of VGC logs will actually expose it. Just for you, here is an example (and don't ask me about allocate/surivor hybrid now - it's very CS specific)

'''    
<mem type="nursery" free="401962224" total="805306368" percent="49">
      <mem type="allocate/survivor" free="401962224" total="402653184" percent="99" />
      <mem type="evacuate" free="0" total="402653184" percent="0" />
</mem>
```

All that to say, you may want to expand this paragraph with a few words to introduce the notion of evacuate, but we can also defer it and expose it only in CS specific documentation.

btw, we also sometime say 'evacuate' instead of 'copied'.

 3. A local GC scavenge process runs and reachable objects are either copied  from *allocate* space (what is referred as evacuate during scavenge time) into the *survivor* space or into the *tenure* area if they have reached *tenure age*.

Reply:
Modified:

"3. A local GC scavenge process runs and reachable objects are either *evacuated* (copied) into the *survivor* space or into the *tenure* area if they have reached *tenure age*."